docs/esp32/quickref.rst

```diff
@@ -102,6 +102,32 @@ Once the network is established the :mod:`socket <usocket>` module can be used
 to create and use TCP/UDP sockets as usual, and the ``urequests`` module for
 convenient HTTP requests.
 
+LAN
+^^^
```

Comment:
this should match other headings at this level, ie use `===`

Reply:
aah, did you intend for this to be a subheading under `Networking`?

Reply:
yes - exactly. The idea was that unter networking there is WLAN and LAN so the user can jump directly to the section

Reply:
ok